src/cli/commands/ls.js

```diff
@@ -125,17 +150,45 @@ export async function buildTree(
   return {trees, count: buildCount(trees)};
 }
 
+export function getParent(key: string, treesByKey: Object) : Object {
+  return treesByKey[key.split('#').slice(0, -1).join('#')];
+}
+
+export function getExtended(ll: boolean, la: boolean, long: boolean) : boolean {
+  return (ll || la || long);
+}
+
+
+export function setFlags(commander: Object) {
+  commander.option('--depth [depth]', 'depth');
+  commander.option('--prod [prod]', 'prod');
```

Comment:
Can you add some descriptions to these options? It's not really obvious what they're supposed to do.